LukasChiama/Banka-App
# #165057407 Build out data storage and CRUD functions

#### What does this PR do?
Build out functions with native node modules for manipulating data
#### Description of Task to be completed?
Develop database models for resources
Have CRUD functions working
#### How should this be manually tested?
After cloning the repo, cd into the server folder and run npm start.
Test endpoints with Postman and see that data is saved by the DB built by this commit
#### Any background context you want to provide?
Since this project will not use a database, this PR delivers a native database with the node.js fs module. This will enable better interactivity with data.
#### What are the relevant pivotal tracker stories?
[#165057407](https://www.pivotaltracker.com/story/show/165057407)
